CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Moved Cloud-J and Fast-JX input directories to Cloud-J and new Fast-JX menus respectively in `geoschem_config.yml`
 - Updated photolysis and aerosol optics input directories to use new mineral dust values in `FJX_scat-aer.dat` and `dust.dat` based on spheroidal shapes
 - Set `State_Diag%Archive_SatDiagn` to true if `State_Diag%Archive_SatDiagnPMID` is true
+- Disabled the `KppTime` diagnostic output the `fullchem_alldiags` integration tests; this will vary from run to run causing difference tests to fail
```

Comment:
I think there is a missing "in"... diagnostic output in the 'fullchem_alldiags"...

Reply:
Now fixed in https://github.com/geoschem/geos-chem/pull/2786/commits/bdf909b3e6c7eb8ffebb7b5830237753af251e32